src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx

```diff
@@ -272,4 +272,7 @@ EXAMPLES
   <data name="InvalidArgument" xml:space="preserve">
     <value>Invalid argument '{0}', did you mean:</value>
   </data>
+  <data name="DeprecatedVersionParameter" xml:space="preserve">
+    <value>Old functionality of '-version 2.0' is deprecated. Now '-version' parameter shows current PowerShell version. </value>
```

Comment:
I think we should change this error message to:

```none
Previous usage of '-version {0}' is not supported.  '-version' currently only supports returning the current PowerShell version.
```

cc @joeyaiello

Reply:
I changed the error message - please review.